celery_slack/callbacks.py

```diff
@@ -70,7 +71,10 @@ def wrapped_func(self, exc, task_id, args, kwargs, einfo):
             """
             attachment = get_task_failure_attachment(self.name, exc, task_id, args, kwargs, einfo, **cbkwargs)
 
-            if attachment:
+            annotations = getattr(inspect.trace()[-1][0].f_locals["task"], "__annotations__", {})
```

Comment:
Is there any easier way to get the annotations using `self` or the `func` passed here ?

Reply:
No, I guess. `self` is the Celery app and `func` is the `on_failure` decorated function. It is tricky, I guess.